vespa/application.py

```diff
@@ -582,6 +581,157 @@ def _handle_result_callback(
                 queue.join()
                 consumer_thread.join()
 
+    def feed_async_iterable(
+        self,
+        iter: Iterable[Dict],
```

Comment:
In case of reserved words, I think it is worth the change.
Confusing for devs.

Reply:
Hmm, since this is a public API, we would need to add deprecation-warning for a while first if we want to rename the keyword arg from `feed_iterable`. That should nevertheless be a separate PR. So, then the question is if this new method should have both the renamed arg, but also `iter` in the meanwhile, to avoid divergent naming. 

Maybe @jobergum could enlighten us a little on best practice here? 🙏

Reply:
Right, backwards compatibility for public APIs.
Separate PR makes sense.

Reply:
Since this is a new method with a new signature we don't need to make the same mistake, it this works well, we can deprecate the sync version.